packages/gui/src/components/offers/OfferSummary.tsx

```diff
@@ -182,9 +182,3 @@ export default function OfferSummary(props: Props) {
     </Flex>
   );
 }
-
-OfferSummary.defaultProps = {
-  isMyOffer: false,
-  imported: false,
-  rowIndentation: 3,
```

Comment:
I see that you just removed default props when it is defined as required in the Props.
But it is not valid as you can see for rowIndentation. 
Please change all required props to optional and use defined default values.

Reply:
Please do it for all other components

Reply:
rowIndentation is not used in <OfferSummary and therefore it will break functionality